### ISSUE TITLE
system.clients is missing information with regards to connected clients

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#6946]
Scylla 4.0.X

```
cat /etc/scylla/scylla.yaml  | grep ^auth
authenticator: PasswordAuthenticator
authorizer: CassandraAuthorizer
```

```
[root@e40a07114766 /]# cqlsh -u cassandra -p cassandra
Connected to  at 172.17.0.3:9042.
[cqlsh 5.0.1 | Cassandra 3.0.8 | CQL spec 3.3.1 | Native protocol v4]
Use HELP for help.
cassandra@cqlsh> select * from system.clients ;

 address    | port  | client_type | connection_stage | driver_name | driver_version | hostname | protocol_version | shard_id | ssl_cipher_suite | ssl_enabled | ssl_protocol | username
------------+-------+-------------+------------------+-------------+----------------+----------+------------------+----------+------------------+-------------+--------------+-----------
 172.17.0.3 | 60618 |         cql |             null |        null |           null |     null |                0 |        0 |             null |        null |         null | anonymous
 172.17.0.3 | 60620 |         cql |             null |        null |           null |     null |                0 |        1 |             null |        null |         null | anonymous
```

- too many fields are null all of them should have values.
- username is incorrect its anonymous it should be cassandra 
- protocol version is incorrect its not version 0

**Comment by tzach** [comment]
@amnonh we can include this info in the connect table in the monitoring dashboard

**Comment by haaawk** [comment]
Can we close this already @jul-stas ?

**Comment by jul-stas** [comment]
The encryption info is missing and the last time I talked to @elcallio (> year ago) I learned that it's not trivial to get this info. We can give it a try, but I'm not proficient with crypto unless I'll have to be.

**Comment by slivne** [comment]
Aside of encryption and hostname all should be populated if this is not the case open a bug. 

pushing these based on requests

**Comment by jul-stas** [comment]
I'm closing this in favor of this: #9216

**Comment by avikivity** [comment]
Usually it's better to close the new bugs, to preserve the historical record.

**Comment by avikivity** [comment]
And it's not possible to backport issues that have no commits attached.

**Comment by jul-stas** [comment]
I edited #7349 by adding "Fixes #6946", but apparently it didn't work (too late?)
What can I do instead?

**Comment by avikivity** [comment]
Ah, then add "Fixed by <commit_hash> here, and add the backport candidate label (mimicking what the robots would do if you added the Fixes # thing in time).

**Comment by jul-stas** [comment]
Fixed by fb24bccf046d319e6f27ed6a6dc0f9079d19d741
Note: needs backporting to 2020 and 2021

**Comment by avikivity** [comment]
Not a bug, not backporting.
